00-HTML-CSS-basics/CSS/Layout 2.5.1/css/style.css

```diff
@@ -0,0 +1,113 @@
+
+/*font styles*/
+/*----------*/
+
+@font-face {
+  font-family: 'MyFont';
+  src:url('../fonts/amatic/Amatic-Bold.ttf');
+  font-weight: bold;
+  font-style: normal;
+  font-size: 14px;
+ 
+}
+
+/*element selectors*/
+/*----------------*/
+
+/*applying box-sizing property in order to stablish same-sized boxes*/
+* {
+  -webkit-box-sizing: border-box;
+     -moz-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+html {
+  font-family: MyFont, Ubuntu, Arial, TimesNewRoman, sans-serif;
+}
+
+
+header{
+  top: 0;
+}
+
+/*Class elements*/
+
+.header {
+  padding-top: 1rem;
+  width: 100%;
+  font-size: 3.286rem;
+  background-color: grey !important; /*Point 6: override inline style*/
+}
+
+
+
+
+.text-alignment-center{
+   text-align: center;
+}
+
+/*Class selectors*/
+/*--------------*/
+/*class attribute that ends with 'r'* (already validated) */ 
+*[class] {
+  display: inline-block;
+}
+
+/*class attribute that ends with 'r'* (already validated) */ 
+*[class$="r"] {
+  background-color: magenta;
+}
+
+/*class attribute containing 'a', which doesn't end with 'r' (already validated) */
+*[class*="a"]*:not([class$="r"]) {
```

Comment:
Not needed. It was left from the previous excersise! I will remove it in the next commit.